app/src/main/java/waiter/RouteDirector.java

```diff
@@ -0,0 +1,54 @@
+package waiter;
+
+public class RouteDirector {
+
+    public Route constructGetSimpleGet(RouteBuilder routeBuilder) {
+        return routeBuilder.newUp()
+                .url("/simple_get")
+                .method("GET")
+                .status("200 OK")
+                .headers("Content-Length:")
+                .body("")
+                .build();
+    }
+
+    public Route constructHeadSimpleGet(RouteBuilder routeBuilder) {
+        return routeBuilder.newUp()
+                .url("/simple_get")
+                .method("HEAD")
+                .status("200 OK")
+                .headers("Content-Length:")
+                .body("")
+                .build();
+    }
+    
+    public Route constructGetSimpleGetWithBody(RouteBuilder routeBuilder) {
+        return routeBuilder.newUp()
+                .url("/simple_get_with_body")
+                .method("GET")
+                .status("200 OK")
+                .headers("Content-Length:")
+                .body("Hello world")
+                .build();
+    }
+
+    public Route constructNoUrl(RouteBuilder routeBuilder) {
+        return routeBuilder.newUp()
+                .url("")
+                .method("")
+                .status("404 Not Found")
+                .headers("Content-Length:")
+                .body("")
+                .build();
+    }
+
+    public Route constructUrlExistsButNoMethodFound(RouteBuilder routeBuilder, String url) {
+        return routeBuilder.newUp()
+                .url(url)
+                .method("")
```

Comment:
Same thing here, do we need this route?

Reply:
We need this route to cover our bases for if someone makes a call to the right url with the wrong or no method, without this the server would break if someone did something like a head request to simple_get_with_body.

Reply:
I see. This also feels strange to me for the same reasons I gave in the other comment.